src/Map/src/Bridge/Leaflet/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/ux-map-leaflet",
```

Comment:
Should be `symfony/ux-leaflet-map` to be consistent with how we are naming bridges in Symfony.